### PR TITLE
Update boss_mennu_the_betrayer.cpp

### PR DESCRIPTION
**Pull request by AnonXS**
https://raw.githubusercontent.com/ACID-Scripts/TBC/14b7e4fe1d1dec0f672f608f526c0fb49a8b90ac/acid_tbc.sql

Resolves: https://bitbucket.org/looking4group_b2tbc/looking4group/issues/1062/mennu-the-betrayer-slave-pens

### CODE REVIEW

File: src/scripts/scripts/zone/coilfang_resevoir/slave_pens/boss_mennu_the_betrayer.cpp [hunk 1]
```diff
@@ -93,15 +93,15 @@ struct boss_mennu_the_betrayerAI : public ScriptedAI
...
+            HealingWard_Timer = urand(15700, 22900);
         }
         else
             HealingWard_Timer -= diff;
 
         if(NovaTotem_Timer < diff)
         {
             AddSpellToCast(m_creature, SPELL_CORRUPTED_NOVA_TOTEM);
-            NovaTotem_Timer = 45000;
+            NovaTotem_Timer = urand(26200, 26200);
```
Comment (Xadras) [inline src/scripts/scripts/zone/coilfang_resevoir/slave_pens/boss_mennu_the_betrayer.cpp]: Hm rand between the same times.. let me guess ... always 26200!

File: src/scripts/scripts/zone/coilfang_resevoir/slave_pens/boss_mennu_the_betrayer.cpp [hunk 2]
```diff
@@ -112,23 +112,23 @@ struct boss_mennu_the_betrayerAI : public ScriptedAI
...
+            LightningBolt_Timer = urand(19300, 29100);
         }
         else
             LightningBolt_Timer -= diff;
 
         if(EarthGrab_Timer < diff)
         {
             AddSpellToCast(m_creature, SPELL_EARTHGRAB_TOTEM);
-            EarthGrab_Timer = 30000;
+            EarthGrab_Timer = urand(26200, 26200);
```
Comment (Xadras) [inline src/scripts/scripts/zone/coilfang_resevoir/slave_pens/boss_mennu_the_betrayer.cpp]: same as above ^^

**Comment by AbaddonL4G** [comment]
Why rand?

**Comment by AnonXS** [comment]
yea taken from acid scripts, didnt think about it. 

But would it work like this? urand(number, samenumber); ?

**Comment by AbaddonL4G** [comment]
Umm y i think but there is no use for it

**Pull request merged**